dms/models/directory.py

```diff
@@ -76,10 +71,22 @@ class DmsDirectory(models.Model):
         index=True,
         store=True,
         readonly=False,
-        compute="_compute_directory_type",
+        compute="_compute_parent_id",
         copy=True,
+        default=lambda self: self._default_parent_id(),
     )
 
+    def _default_parent_id(self):
+        context = self.env.context
+        if context.get("active_model") == self._name and context.get("active_id"):
+            return context.get["active_id"]
+        else:
+            return (
+                self.env["dms.directory"]
+                .search([("is_root_directory", "=", True)], limit=1)
+                .id
+            )
```

Comment:
🤔 Do you think this makes any sense?

I mean... **by default** every new directory should be created as a children of some random root directory? Wouldn't it make sense to leave it as `False` instead?

This is my suggestion:

```suggestion
        return False
```

---

If I'm right and this must be changed, please take that chance to put fields first and then, later, methods. At least with the new ones introduced. This follows Odoo guidelines (upon which OCA's are based): https://www.odoo.com/documentation/13.0/reference/guidelines.html#symbols-and-conventions